communications/slack-culture.rst

```diff
@@ -45,28 +45,103 @@ Summary
 
 - Be professional and courteous in informal channels. (:ref:`Details<slack-informal-channels>`)
 
+Always consult a channel's description for channel-specific rules of engagement.
```

Comment:
Could we get some guidance on what goes into Topics and what goes into the Description? Also recommending that one of them, or both, be present for all channels would be great.